docs/source/blocks/settings.md

```diff
@@ -241,6 +241,24 @@ const initialBlocks = {
 
   If the choice is limited to one item, then the setting hides itself from the `listing` block settings list.
 
+## Video block configuration
+
+This section describes how to configure video blocks.
+Although only Peertube is documented here, you can use this example for other video providers.
+
+### Peertube instances
+
+You can configure the video block to use other instances of Peertube by adding them to the `allowedPeertubeInstances` array in your project configuration object, as shown in the following example.
```

Comment:
The example shows assigning an array to a variable, not to the config settings. Does it go at the top level? Does it go somewhere in the blockConfig? This documentation doesn't tell me, so I guess I have to go look at the source code...

Reply:
I agree, but there is a section about [configuring blocks](https://6.docs.plone.org/volto/blocks/settings.html#configuring-a-new-block) for the how to, and the other examples like [groupBlocksOrder](https://6.docs.plone.org/volto/blocks/settings.html#groupblocksorder-the-blocks-chooser-group-order) and [initialBlocks](https://6.docs.plone.org/volto/blocks/settings.html#initialblocks-initial-blocks-per-content-type) which are identical to this one.

Isn't this a more general blocks configuration documentation work?

Reply:
I think it would be clearer to show it as 
```
config.blocks.blocksConfig.video.allowedPeertubeInstances = [
  'freediverse.com', // About: https://freediverse.com/about/instance
  'fair.tube', // About: https://fair.tube/about/instance
  'tube.reseau-canope.fr', // About: https://tube.reseau-canope.fr/about/instance
  'peertube.eus', // About: https://peertube.eus/about/instance
];
```

(And I think the other examples on the page would also be clearer if they actually showed where in the config it goes, but we don't have to fix them here.)